src/core/jpg.js

```diff
@@ -363,6 +363,12 @@ var JpegImage = (function jpegImage() {
       // find marker
       bitsCount = 0;
       marker = (data[offset] << 8) | data[offset + 1];
+      // Some bad images seem to pad Scan blocks with zero bytes, skip past
+      // those to attempt to find a valid marker (fixes issue4090.pdf).
+      while (marker === 0xFF && offset < data.length - 1) {
```

Comment:
How about this instead, to better align the code with the comment and also to support multiple zero bytes?
```javascript
while (data[offset] === 0x00 && offset < data.length - 1) {
  ...
```
Or, I suppose that we could just do:
```javascript
if (data[offset] === 0x00 && offset < data.length - 1) {
  ...
```
Which one do you prefer?